docs/getting-started/what-is-dotnet.md

```diff
@@ -133,18 +139,17 @@ The .NET Framework on the Windows desktop allows you to create console applicati
 
 - Console applications can be compiled to run cross-platform on Windows, the Macintosh, and Linux.
 
-- Console applications can be compiled natively. This combines the benefits of a managed application with the performance of a native C/C++ native application. 
+- Console applications can be compiled natively. This combines the benefits of a managed application with the performance of a native C/C++ application. 
 
 ## Ease of Installation ##
 
 Because .NET Core is not an operating system component, installation does not require administrator privileges, and it does not touch system components like operating system directories or, on Windows systems, the Windows Registry. Installation is as simple as copying some files to a directory on the target computer or compiling the framework natively into the application. 
 
-Moreover, the particular set of 
 New developers can get started developing for .NET in under one minute, including downloading the frameworks and tools. You can install .NET Core from the [.NET Core getting started page](https://dotnet.github.io/getting-started/#/windows ".NET Core getting started page"). 
```

Comment:
marketing - yuck
New developers can get started developing for .NET very quickly.,You can install .NET Core ...